src/controller/java/AndroidDeviceControllerWrapper.h

```diff
@@ -85,6 +93,11 @@ class AndroidDeviceControllerWrapper : public chip::Controller::DevicePairingDel
 
 private:
     using ChipDeviceControllerPtr = std::unique_ptr<chip::Controller::DeviceCommissioner>;
+    chip::Crypto::P256Keypair mIssuer;
+        bool mInitialized  = false;
+        uint32_t mIssuerId = 0;
+        uint32_t mNow      = 0;
+        uint32_t mValidity = 365 * 24 * 60 * 60 * 10;
```

Comment:
Indentation is wrong here.

Also, why is there an mValidity here in some delegate? Whyis state keeping here?